orbit-2-core/src/main/java/com/babylon/orbit2/Container.kt

```diff
@@ -42,6 +74,12 @@ interface Container<STATE : Any, SIDE_EFFECT : Any> {
             }
     }
 
+    /**
+     * Represents additional settings to create the container with.
+     *
+     * @property sideEffectCaching Whether side effect caching should be turned on or off.
```

Comment:
doesn't really explain what side effect caching is, better phrased as "When set to true the last side effect is emitted for every subscriber" (or whatever makes sense)